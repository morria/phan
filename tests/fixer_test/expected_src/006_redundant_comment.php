<?php
namespace ns6;
use function rand;
use function var_export;
use function ns6\returns_int;

function returns_int() : int {
    return rand(0, 10);
}
var_export(returns_int());

$doubles_input =
    function (int $input) : int  {
        return $input * 2;
    };
var_export($doubles_input(21));

class C {
    public static function f($value) : void {
        var_export($value);
    }
}

C::f('F');
