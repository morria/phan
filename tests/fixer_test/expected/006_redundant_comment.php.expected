src_copy/006_redundant_comment.php:8 PhanPluginRedundantFunctionComment Redundant doc comment on function returns_int(): '/** @return int */'
src_copy/006_redundant_comment.php:18 PhanPluginRedundantClosureComment Redundant doc comment on closure Closure(int $input) : int: "/**\n * @param int \$input\n * @return int\n */"
src_copy/006_redundant_comment.php:27 PhanPluginRedundantMethodComment Redundant doc comment on method f(): "/**\n     *\n     */"
